app/views/loans/viewloanaccountdetails.html

```diff
@@ -897,8 +897,8 @@ <h3 class="bolder">{{'label.heading.confirm' | translate}}</h3>
                             <div class="form-group">
                                 <div class="col-sm-12 col-md-12">
                                     <textarea rows="1" class="form-control marginbottom0px" id="textarea"
-                                              placeholder="{{'label.input.enternote' | translate}}" ng-model="formData.note"></textarea>
-
+                                              placeholder="{{'label.input.enternote' | translate}}" ng-model="formData.note" style="min-width: 1050px; max-width: 1050px; "></textarea>
```

Comment:
What is the use of styling here ?

Reply:
If we don't use max-width here, the text box can go outside the div container

Reply:
Did you tried in local ? I don't think it will go outside, if it goes please post the screenshot here

Reply:
![capture_2017_10_04_21_20_33_708](https://user-images.githubusercontent.com/20682192/31185607-03a37c14-a94a-11e7-8365-34dbbea8a0b3.png)
Here you can see

Reply:
I see you have added some style there in Inspector! 

This is what i see pulling your code and removing the styling in `text-area`
![selection_066](https://user-images.githubusercontent.com/9936881/31185905-cf671220-a94a-11e7-9a3c-23ff4723ff00.png)

Reply:
So probably the fix would be removing that styling from `text-area`

Reply:
No, I haven't added the these style, these are default by firefox (the height and width of text-area).  When increasing the width, it goes outside of <div class="card">

Reply:
![capture](https://user-images.githubusercontent.com/20682192/31187768-7952dfa8-a950-11e7-92ec-aba493e6cf0b.PNG)
With my fix user is restricted to 1050 px width and user can't extend it beyond the div element

Reply:
![capture2](https://user-images.githubusercontent.com/20682192/31187930-f52c447a-a950-11e7-91c0-d76284ae06b0.PNG)
And without my fix user can extend it beyond the div element and then cannot resize it

Reply:
Thanks @luckyman000786